priv/templates/channel/channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule <%= module %>Channel do
+  use <%= base %>.Web, :channel
+
+  def join("<%= plural %>:lobby", _auth_message, socket) do
```

Comment:
We usually scope to resource id, or some 'top level' thing, i.e. "lobby", including ":join" as part of the topic doesn't make sense?